<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_window</class>
 <widget class="QMainWindow" name="admin_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>admin_window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BookRegButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Book registration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BookOpButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Borrow/Return operation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BookSearchButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CardManageButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Library card management</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
